helm/hpcc/templates/_helpers.tpl

```diff
@@ -615,3 +615,44 @@ resources:
     memory: "100M"
 {{- end -}}
 
+
+{{/*
+Create placement related settings
+Pass in dict with placement
+*/}}
+{{- define "hpcc.doPlacement" -}}
+{{- if .placement.nodeSelector }}
+nodeSelector:
+{{ toYaml .placement.nodeSelector | indent 2 }}
+{{- end -}}
+{{- if .placement.tolerations }}
+tolerations:
+{{ toYaml .placement.tolerations }}
+{{- end -}}
+{{- if .placement.affinity }}
+affinity:
+{{ toYaml .placement.affinity }}
+{{- end -}}
+{{- if .placement.schedulerNames -}}
+{{- range $profileName := .placement.schedulerNames }}
+schedulerName: {{ $profileName }}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Check if there is any placement configuration
+Pass in dict with root, pod, cluster and type
+*/}}
+{{- define "hpcc.placementMatchName" -}}
+{{- if .root.Values.placement }}
+{{- $pod := .pod -}}
+{{- $cluster := .cluster -}}
+{{- $type := .type -}}
+{{- range $placement := .root.Values.placement -}}
+{{- if or (has $pod $placement.pods) (has $cluster $placement.pods) (has $type $placement.pods) -}}
+{{ include "hpcc.doPlacement" (dict "placement" $placement) -}}
```

Comment:
"thor" is probably confusion. It can mean "type" or "cluster". I wonder we should recommend name convention for cluster name. So instead of "thor" probably better "thor1", thor2", etc. I will edit or remove ["thor"] sample which actually just for my test